### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by dutterbutter**
## Changes
- Updated Travis badge to point to development instead of master
- Updated install field 
- 

## Tests:
```

```

### Fixes:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 [![GoDoc](https://godoc.org/github.com/ChainSafeSystems/gossamer?status.svg)](https://godoc.org/github.com/ChainSafeSystems/gossamer)
 [![Go Report Card](https://goreportcard.com/badge/github.com/ChainSafeSystems/gossamer)](https://goreportcard.com/report/github.com/ChainSafeSystems/gossamer)
-[![](https://img.shields.io/travis/com/ChainSafeSystems/gossamer/master.svg?label=master&logo=travis "Master Branch (Travis)")](https://travis-ci.com/ChainSafeSystems/gossamer)
+[![Build Passing](https://img.shields.io/travis/com/ChainSafeSystems/gossamer/development.svg?label=development&logo=travis "Development Branch (Travis)")](https://travis-ci.com/ChainSafeSystems/gossamer)
```
Comment (ansermino) [inline README.md]: Can we also add master?

**Comment by dutterbutter** [comment]
@noot @ansermino the Travis build failed again on that `TestSend` from p2p?

**Comment by noot** [comment]
@dutterbutter looks like ping is failing not send... thought I fixed that but guess not. will look into it

**Comment by ansermino** [comment]
should I override and merge this? @noot @dutterbutter

**Comment by dutterbutter** [comment]
> should I override and merge this? @noot @dutterbutter

It is just the readME so should be fine, I just re-ran Travis and it has passed.

**Comment by noot** [comment]
@ansermino yes probably, I'm not sure why travis is suddenly failing on tests it passed on before

**Pull request merged**